### PR TITLE
Update support v1 client

### PR DESCRIPTION
**Pull request by lukechurch**
@kasperpeulen This updates the JSON and v1 client to support completion requests

@devoncarew The commands send to the discovery tool seems to have changed - from generate -> package. Can you confirm?

### CODE REVIEW

File: doc/generated/v1.dart [hunk 1]
```diff
@@ -603,12 +650,145 @@ class DocumentResponse {
 }
 
 
-/** Not documented yet. */
+class Edit {
```
Comment (devoncarew) [inline doc/generated/v1.dart]: This is a very short, fairly generic class name. Perhaps consider renaming the source class this is generated from (`FixEdit`? `SourceEdit`?)

File: doc/generated/v1.dart [hunk 2]
```diff
@@ -603,12 +650,145 @@ class DocumentResponse {
...
+    var _json = new core.Map();
+    if (fixes != null) {
+      _json["fixes"] = fixes.map((value) => (value).toJson()).toList();
+    }
+    return _json;
+  }
+}
+
+
+class ProblemFix {
```
Comment (devoncarew) [inline doc/generated/v1.dart]: Some docs on the source class would help here, I'm not 100% what a problem fix is.

File: tool/grind.dart [hunk 3]
```diff
@@ -39,7 +39,7 @@ discovery(GrinderContext context) {
   // Generate the Dart library from the json discovery file.
   Pub.global.activate(context, 'discoveryapis_generator');
   Pub.global.run(context, 'discoveryapis_generator:generate', arguments: [
-    'generate',
+    'package',
```
Comment (devoncarew) [inline tool/grind.dart]: Yup, @lukechurch, this did change. I didn't realize that they had published the new version. Also, slightly prettier output :)

**Comment by lukechurch** [comment]
Yeah, these things do end up being somewhat torn out of context.

I'll rename to more specific and document.

**Comment by devoncarew** [comment]
lgtm! Yup, the generator's command changed. It can also generate stand-alone libraries now.

**Pull request merged**